code/modules/mob/living/carbon/human/species_types/monkeys.dm

```diff
@@ -25,11 +25,10 @@
 	)
 	inherent_traits = list(
 		TRAIT_CAN_STRIP,
-		TRAIT_VENTCRAWLER_NUDE,
+		TRAIT_GUN_NATURAL,
 		TRAIT_PRIMITIVE,
+		TRAIT_VENTCRAWLER_NUDE,
 		TRAIT_WEAK_SOUL,
-		TRAIT_GUN_NATURAL,
-		//TRAIT_LITERATE, monkeys shouldn't be able to read or write
```

Comment:
This is a code comment to show that the decision to not make them literate was deliberate.

Reply:
Reverted my change then.